### ISSUE TITLE
Can't edit model

### ISSUE THREAD

**Issue by lloydpick** [manyfold3d/manyfold#1778]
Updated to the latest (v0.55.0), can't edit any model, changes just don't get applied. Tried changing the model name, the URL, memo, nothing goes through. Logs also show no errors.

Advanced admin is also throwing errors... `NotImplementedError (You must define #resolve in ApplicationPolicy::Scope)`. Only the dashboard and the logs page function.

**Comment by Floppy** [comment]
Thanks @lloydpick I've verified this on my own install, and will get a fix out asap. Closing this as a duplicate of #1777, which reports the same problem.

**Comment by Floppy** [comment]
Actually reopening this, as I think the "can't edit models" is a separate issue.

**Comment by Floppy** [comment]
So, I can't replicate the model save problem on my local environment, my personal instance, or on https://try.manyfold.app, which is odd.

**Comment by Floppy** [comment]
@lloydpick do other things get saved? Creators, collections, file options?

**Comment by ToasterUwU** [comment]
Im having the same issue of not having my changes being applied.
Also only after updating.

I can however edit it by using a different part of the interface. So if I click the the name in the top instead of using the edit menu, rewrite the name, click of it and reload the tab, the named stays changed.

Im not seeing anything in the log that would relate to this.

**Comment by Floppy** [comment]
Thanks for confirming ToasterUwU. Could I ask either of you to try and replicate it on https://try.manyfold.app? You should be able to edit in the same way. I think I must be doing something different to you - if two of you have it, it's definitely a thing!

**Comment by Floppy** [comment]
Ignore that, I see it now. I think I must have got confused earlier. Looking into it.

**Comment by Floppy** [comment]
Ah, OK, that's what happened. Editing https://try.manyfold.app/libraries/1/models/2 works fine, but https://try.manyfold.app/libraries/1/models/3 doesn't save anything. I suspect there's a validation failing, and it's silently ignoring the edit. Which is bad. Will fix.

**Comment by Floppy** [comment]
I suspect this is to do with licenses. I also can't set the license on the Bunny back to nothing, so I reckon something's happening there.

**Comment by ToasterUwU** [comment]
That makes sense. It started after the license thing got added, and I don't have any models with licenses, so I had the issue with all of them naturally.

**Comment by Floppy** [comment]
Whereas I went and tested that licenses worked, so missed it! 😬

**Comment by Floppy** [comment]
Turns out, it all works perfectly well, and is well tested, for the case when you set the license to `nil`. Unfortunately, the value that comes back from the form is `""`. That isn't quite so valid.

**Comment by ToasterUwU** [comment]
true that lol, happens to the best of us

**Comment by Floppy** [comment]
This should now be fixed and is in release v0.55.2. Fingers crossed!
